packages/psss/src/views/AlertsOutbreaksView.js

```diff
@@ -56,7 +64,7 @@ export const AlertsOutbreaksView = () => {
     <>
       <Header Title={Title} ExportModal={ExportModal} />
       <TabsToolbar links={links} />
-      <AlertsRoutes />
+      <AlertsTabView />
```

Comment:
In the designs the Archive view is quite different to the Alerts view. There are different columns in the table, different actions in the table, different page layout (full width vs. 2 columns), filters etc. At the moment the pages are the same. Is this just temporary or has there been a design change? I still lean to the idea of different templates if the views are different but I am open to combining them if you think the pros out weight the costs.